src/app/features/registries/components/justification-review/justification-review.component.html

```diff
@@ -21,13 +24,22 @@ <h4 class="mb-2">{{ 'registries.justification.updatedList' | translate }}</h4>
       <p class="font-italic text-red-500">{{ 'registries.justification.noUpdates' | translate }}</p>
     }
   </p-card>
+
   @for (page of pages(); track page.id) {
     <p-card class="w-full">
       <h2 class="mb-4">{{ page.title }}</h2>
 
       @if (page.description) {
         <p class="mb-4">{{ page.description }}</p>
       }
+
+      @if (page.questions?.length) {
+        <osf-registration-blocks-data
+          [questions]="page.questions"
+          [reviewData]="schemaResponseRevisionData()"
+        ></osf-registration-blocks-data>
+      }
+
       @if (page.sections?.length) {
```

Comment:
Note for posterity, this addresses an issue where certain questions were not showing up in the overview page. This was stemming from the fact that there was an implicit assumption in how we were rendering forms before where we assumed the blocks would show up in an order like PageHeading -> SectionHeading -> QuestionLabel, but that is not necessarily the case